mypyc/lib-rt/CPy.h

```diff
@@ -716,6 +727,25 @@ static inline char CPyDict_CheckSize(PyObject *dict, Py_ssize_t size) {
     return 1;
 }
 
+// Unsafe because it assumes dict is actually a dict.
```

Comment:
I'm not entirely sure why this goes here, but I just stuck it with the other version of the same